examples/cpp/aot_inductor/bert/README.md

```diff
@@ -50,6 +50,7 @@ mv bertcppaot model_store/
 Start torchserve using the following command
 
 ```
+export LD_LIBRARY_PATH=`python -c "import torch;from pathlib import Path;p=Path(torch.__file__);print(f\"{(p.parent / 'lib').as_posix()}:{(p.parents[1] / 'nvidia/nccl/lib').as_posix()}\")"`:$LD_LIBRARY_PATH
```

Comment:
Yes, its in the main README too. I just put it here so people don't forget. We can't rely on build.sh because people will not always rebuild before running. My plan is to figure out the path in the frontend but I don't want to mess around there so close to the release. As this is experimental, we should be fine with this solution for this release and the next will automatically set the path.